recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ about:
   #
   # https://github.com/hackebrot/pytest-cookies/pull/10
   #
```

Comment:
This comment can be dropped as well. Was just referencing the removed license. When removing it feel free to add `[ci skip]` in the new commit message as we already know this passes and the change is non-actionable for the CI.